.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -59,6 +59,11 @@ jobs:
             warning: "Dies ist eine unveröffentlichte Version auf GitHub"
             sections: "sections/translated/de"
             luaengine: true
+          - language: cs
+            version: Verze
+            warning: "Toto je nepublikovaná verze vyrobená z commitu na GitHubu"
+            sections: "sections/translated/cs"
```

Comment:
```suggestion
```
Actually, after recent changes, this line is no longer required. Can you fix it in #555?

Reply:
Do you have a PR where this was introduced? Just for a reference

Reply:
Not sure why you would need that? Just delete those lines.

Reply:
Just to understand what I'm doing :)